src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -384,24 +384,7 @@ public string TransferEncoding
         }
 
 
-        public bool KeepAlive
-        {
-            get
-            {
-                return _webHeaderCollection[HttpKnownHeaderNames.KeepAlive] == bool.TrueString;
-            }
-            set
-            {
-                if (value)
-                {
-                    SetSpecialHeaders(HttpKnownHeaderNames.KeepAlive, bool.TrueString);
-                }
-                else
-                {
-                    SetSpecialHeaders(HttpKnownHeaderNames.KeepAlive, bool.FalseString);
-                }
-            }
-        } 
+        public bool KeepAlive { get; set; } = true;
```

Comment:
> I don't understand this fix.

What don't you understand?

We track whether KeepAlive is set, in a field, as is done on desktop.  Then when we create the request, we set the appropriate header based on KeepAlive.